### ISSUE TITLE
5800X3D Guide worked!

### ISSUE THREAD

**Issue by briankelly** [PrimeO7/How-to-undervolt-AMD-RYZEN-5800X3D-Guide-with-PBO2-Tuner#11]
Thanks for writing this. On first run I was able to improve my 10-minute CB23 MC score from 13,589 to 14,068 (11 passes). This was going from stock 5800X3D to -10 offset on all cores.

I'll play around with this a bit more to see if a larger offset produces better results. The temps were actually higher with the -10 offset, but the cores stayed at 4.3 GHz all-core during the CB23 workload.

Here's a screenshot from [LibreHardwareMonitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) showing voltage, power, temps, and clocks from my 5800X3D over two, 10-minute CB23 runs.

![image](https://user-images.githubusercontent.com/306395/205113769-d65c9b34-16bd-4914-acf3-2fa8c0a04059.png)

**Comment by briankelly** [comment]
For anyone who's curious, here are the results after running a number of additional 10-minute CB R23 tests.

![Screenshot 2022-12-01 152500](https://user-images.githubusercontent.com/306395/205156802-2eb34e1f-287d-4967-bef5-4faba0c239b6.png)

**Comment by Diceairza** [comment]
Your Temps is a bit high and score is a bit low. What was your ambient temps.  I did -30 and my cinebench r23 is just over 15k like 15023 with temps at 70C max stable so far in games and normal web browsing. CPU boost to 4450 all core as well. 

It's summer here in South Africa and ambient temps is about 25-30C

This is the pc specs.

p400a
arctic p12 fans (1 rear, 1 top rear, 3 front)
Deepcool ag620 cpu cooler
32GB Corsair vengance RGB SL 3600mhz ram
Galax hof rtx 3080 ti
windows 11

**Comment by briankelly** [comment]
Thanks for sharing your numbers, @Diceairza. My ambients are low, 19-22 C.

I'm using a good cooler, Arctic Liquid Freezer II 280 mm, but perhaps I need even larger than 280mm radiator (with 2x140mm fans), to keep the chip cool.

Are your >15k scores in cinebench R23 over a single run, or over a longer duration (10-30 minutes)?

**Comment by Diceairza** [comment]
> Thanks for sharing your numbers, @Diceairza. My ambients are low, 19-22 C.
> 
> I'm using a good cooler, Arctic Liquid Freezer II 280 mm, but perhaps I need even larger than 280mm radiator (with 2x140mm fans), to keep the chip cool.
> 
> Are your >15k scores in cinebench R23 over a single run, or over a longer duration (10-30 minutes)?

I must admit it's over a single run, but I find it funny that your cooler doesn't do as good as a deepcool ag620. I've just installed this yesterday and tested and was suprised how cool that chip runs but hey silicon lottery. Can also be the motherbord (gigabyte aorus pro v2) that has some good vrm cooling so more efficient voltage to cpu.  what is your complete system specs with case layout maybe i can help a bit.

**Comment by briankelly** [comment]
> what is your complete system specs with case layout maybe i can help a bit.

I appreciate the offer!

- Board: [ASRock X570S PG Riptide](https://www.asrock.com/mb/AMD/X570S%20PG%20Riptide/index.asp#Specification)
- 32 GB memory: 2x16 GB G.SKILL Trident NEO kit (DDR4-3600 CL16) [F4-3600C16D-32GTZN](https://www.gskill.com/specification/165/326/1562839473/F4-3600C16D-32GTZN-Specification)
- Case: [Corsair 110R mid-tower ATX](https://www.corsair.com/us/en/Categories/Products/Cases/Mid-Tower-ATX-Cases/110R-Tempered-Glass-Mid-Tower-ATX-Case/p/CC-9011183-WW#tab-tech-specs)
- CPU Cooler: [Arctic LFII 280](https://www.arctic.de/us/Liquid-Freezer-II-280/ACFRE00066B) with rad on the front of case with 2x140mm fans pulling air into case.
- 140mm exhaust on top of case (Arctic P14 PWM)
- 120mm exhaust in rear of case (Arctic P12 PWM)

Case temps should be good. I've run it with and without the side panel on. I have an ASUS TUF 3090 GPU, which heats things up while gaming, but is mostly idle during the CB R23 runs.

I have messed around with memory timings quite a bit, so I welcome feedback on those. Currently, I'm running this G.SKILL B-die kit as 14-14-14-14-28 (tCL, tRCDRD, tRCDWR, tRP, tRAS) at 1.45V with a few secondary timings set. It is stable, but the voltage is on the high side.

Here are a few BIOS screenshots.
![221215130531](https://user-images.githubusercontent.com/306395/207940418-cf45a73a-3e03-4456-83b2-18be046909d5.png)

I've read that this board sets its SoC voltage on the higher side (1.1v). Might be something to change..
![221215130547](https://user-images.githubusercontent.com/306395/207940671-67ef69b2-1a19-4df3-8d6e-e92d7d5bf0ad.png)

Voltage/LL
![221215130722](https://user-images.githubusercontent.com/306395/207941497-b87ae630-6747-4f5c-9fc2-534069be15cf.png)

CPU config
![221215130734](https://user-images.githubusercontent.com/306395/207940344-88b7e8bd-3063-41b8-bb9d-f226622bc690.png)

AGESA version
![221215130811](https://user-images.githubusercontent.com/306395/207940273-4db319e4-cbaa-4b65-abc8-efcbb80bbb85.png)

LCLK / CPU options (all Auto/defaults)
![221215130817](https://user-images.githubusercontent.com/306395/207941685-97c1c2ab-d42e-4dba-9941-ceda7238dd24.png)
![221215130821](https://user-images.githubusercontent.com/306395/207941700-f2bf83da-7aae-4666-a678-e56b15b21595.png)
![221215130917](https://user-images.githubusercontent.com/306395/207941748-ca5a6e71-471b-4c97-b66b-b43901de2a65.png)

I did notice even when in BIOS my CPU idle temps are high.
![221215130947](https://user-images.githubusercontent.com/306395/207941802-5abeaf8c-9fc8-4472-a7db-ae5a80e06ba3.png)


JEDEC/XMP info about memory
![221215130620](https://user-images.githubusercontent.com/306395/207941085-c1163e4a-d6e3-4cd8-88e2-03f99a558930.png)

Timings
![221215130639](https://user-images.githubusercontent.com/306395/207941123-238de019-71a2-4d82-b221-2661e3034c87.png)
![221215130647](https://user-images.githubusercontent.com/306395/207941137-47b851cf-a5c7-4f65-86e0-37fb5ea9c4d8.png)
![221215130712](https://user-images.githubusercontent.com/306395/207941238-c1ef1fe7-b709-49d4-a18a-81b31ade5696.png)

**Comment by briankelly** [comment]
FWIW, I ran one multi-core test and one single-core test after pulling these screenshots. I closed all apps other than my monitoring tool (LHM) and set the Cinebench.exe process to _Above normal_ priority (instead of the default, _Below priority_).

Here are my single-run results with -30 on all cores set:
- CPU (Multi Core): **15,110 pts**
- CPU (Single Core): **1,482 pts** (10.20x MP ratio)

CCD1 Tdie reached a max of 77 deg C during the MC test.

![single-run CB23 5800X3D](https://user-images.githubusercontent.com/306395/207953046-670981f8-d1c4-4469-937c-c9dc85d06e54.png)
